File: 프로그래머스/lv0/120821. 배열 뒤집기/배열 뒤집기.js
function solution(num_list) {
    var reverse = [];
    let count  = num_list.length
    
    for (let i=0; i<count; i++){
        let tmp = num_list.pop()
        reverse.push(tmp)
    }
    return reverse;
    // const solution = (num_list) => num_list.reverse
}

File: 프로그래머스/lv0/120821. 배열 뒤집기/배열 뒤집기.cpp
#include <string>
#include <vector>

using namespace std;

vector<int> solution(vector<int> num_list) {
    vector<int> answer;
    int size = num_list.size();
    for (int i=0; i<size; i++){
        int tmp = num_list.back();
        num_list.pop_back();
        answer.push_back(tmp);
    }

    return answer;
}